src/app/modules/layout/admin-layout/admin-layout.component.html

```diff
@@ -40,7 +40,7 @@
               class="hostname-label"
               matTooltipPosition="above"
               [matTooltip]="'Hostname: {hostname}' | translate: { hostname }"
-            >{{ hostname }}</div>
+            >{{ 'hostname' }}</div>
```

Comment:
that's incorrect. We don't need to touch hostname on the sidenav panel. You made it static 'hostname' text instead of real value.

How it should be: (as previous)
<img width="240" alt="Screenshot 2025-03-15 at 10 16 49" src="https://github.com/user-attachments/assets/de751b6d-e069-4e22-868e-da21391030c0" />

Reply:
@RehanY147 - you haven't fixed it